cmd/istioctl/main.go

```diff
@@ -28,8 +32,22 @@ import (
 
 	"istio.io/manager/cmd"
 	"istio.io/manager/model"
+
+	// Note that to get this package I had to do
+	//     cd $GOPATH/src/k8s.io/client-go
+	//     git checkout v2.0.0
+	// because the default branch that `go get` uses couldn't find this package.
+	"k8s.io/client-go/pkg/util/yaml"
 )
 
+type MetadataAndRule struct {
+	Type string
+	Name string
+	// Spec istio.proxy.v1alpha.config.RouteRule
+	Spec interface{}
+	Rule proto.Message
```

Comment:
If we know it's a rule, we don't have to name the struct accordingly. I suggest renaming this to something generic like InputDoc